package-release.sh

```diff
@@ -8,15 +8,17 @@ fi
 DXVK_VERSION="$1"
 DXVK_SRC_DIR=`dirname $(readlink -f $0)`
 DXVK_TMP_DIR="/tmp/dxvk-$DXVK_VERSION"
-DXVK_ARCHIVE_PATH="$2/dxvk-$DXVK_VERSION.tar.gz"
+DXVK_ARCHIVE_PATH=$(realpath "$2")"/dxvk-$DXVK_VERSION.tar.gz"
 
 function build_arch {
   cd "$DXVK_SRC_DIR"
 
   meson --cross-file "$DXVK_SRC_DIR/build-win$1.txt"  \
         --buildtype "release"                         \
         --prefix "$DXVK_TMP_DIR/install.$1"           \
+	--unity on                                    \
         --strip                                       \
+        -Dskip_tests=true                             \
```

Comment:
`-Dbuild_tests=false` if you change the option. Obvious, but this is the kind of line that I would almost certainly forget to change.